content/en/blog/_posts/2022-12-06-fsgroup-on-mount.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Providing pod's FSGroup to CSI drivers during mount"
+date: 2022-12-06
```

Comment:
```suggestion
date: 2022-12-23
```

Please update the path `content/en/blog/_posts/2022-12-06-fsgroup-on-mount.md` to match.